docs/pages/server-access/guides/openssh.mdx

```diff
@@ -340,6 +340,71 @@ authenticate the host via the certificate we generated earlier.
 
 </Details>
 
+<Details title="Using Proxy Jump in Trusted Clusters?">
+  
+Normally, you connect to leaf cluster SSH nodes through the root cluster proxy,
+which is provided in the `ProxyCommand` line of the SSH config generated above.
+
+In scenarios where leaf proxies are reachable by SSH clients, you might prefer
+to connect directly through the leaf proxies to avoid potential latency issues
```

Comment:
I think we can make "to avoid potential latency issues of the connection being routed through the root cluster" a bit more concise by replacing it with "to achieve lower latency".